lib/schema/schema.go

```diff
@@ -644,6 +644,7 @@ const manifestSchema = `
       "additionalProperties": false,
       "properties": {
         "baseImage": {"type": "string"},
+        "allowPrivileged": {"type": "boolean"},
```

Comment:
Did you try updating with this flag on? I remember `additionalProperties: false` being the culprit for update failures as the existing cluster will probably fail to validate the new application's manifest.

Reply:
@a-palchikov Good call, yeah - it didn't work obviously due to this schema issue.

I took a stab at fixing it, check out my latest commit: https://github.com/gravitational/gravity/pull/620/commits/561f1b3928dfd204f149e3f6d1f6be9032cfcf83. With these changes the upgrade works (I tried 6.0 to this new version with privileged flag enabled).